fv3gfs-physics/fv3gfs/physics/testing/__init__.py

```diff
@@ -1,2 +0,0 @@
-from . import translate_physics
-from .translate_physics import TranslatePhysicsFortranData2Py
```

Comment:
Why were these removed?

Reply:
Since we no longer need to carry separate translate class infrastructure, we don't need this in physics anymore. If we decide to add `TranslatePhysics`, it will go here.